mesonbuild/coredata.py

```diff
@@ -241,6 +260,7 @@ def validate_option_value(self, option_name, override_value):
         raise MesonException('Tried to validate unknown option %s.' % option_name)
 
 def load(filename):
+    global version
```

Comment:
We were using the global `version` implicitly. This just makes it explicit where it's coming from.